superagi/tools/code/write_code.py

```diff
@@ -70,8 +70,9 @@ def _execute(self, code_description: str) -> str:
         logger.info(prompt)
         messages = [{"role": "system", "content": prompt}]
 
-        total_tokens = TokenCounter.count_message_tokens(messages, self.llm.get_model())
-        token_limit = TokenCounter.token_limit(self.llm.get_model())
+        organisation = Agent.find_org_by_agent_id(self.toolkit_config.session, agent_id=self.agent_id)
+        total_tokens = TokenCounter().count_message_tokens(messages, self.llm.get_model())
+        token_limit = TokenCounter(session=self.toolkit_config.session, organisation_id=organisation.id).token_limit(self.llm.get_model())
```

Comment:
initiate single TokenCounter and call both functions.

Reply:
done